installer/pkg/components/kubestate-metrics/prometheusrule.go

```diff
@@ -0,0 +1,91 @@
+package kubestateMetrics
+
+import (
+	"github.com/gitpod-io/observability/installer/pkg/common"
+	"github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func prometheusRule() []runtime.Object {
```

Comment:
I thought we were going to import the prometheusRules from YAML for now. Did I miss something?

Reply:
You're right, I forgot. Although it wasn't a huge amount of work, if annoying